### ISSUE TITLE
non--bare code fails "strict" requirement

### ISSUE THREAD

**Issue by JohnDenker** [atsepkov/RapydScript#92]
#### Expected and Desired Behavior
```
bin/rapydscript --bare  /dev/null | nodejs | wc
      0       0       0
```
#### Observed Behavior (without the --bare)
```

bin/rapydscript  /dev/null | nodejs
[stdin]:1
(function(){"use strict";var ՐՏ_Temp;symbolfor=function ՐՏ_symbolfor_polyfill(
                                              ^
ReferenceError: symbolfor is not defined
    at [stdin]:1:47
    at [stdin]:1:2481
    at Object.exports.runInThisContext (vm.js:74:17)
    at Object.<anonymous> ([stdin]-wrapper:6:22)
    at Module._compile (module.js:460:26)
    at evalScript (node.js:431:25)
    at Socket.<anonymous> (node.js:164:11)
    at Socket.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)
```

Note that removing the "use strict" from the output makes the symptom go away.

Note that there are other manifestations of the same problem, i.e. other symbols that are introduced in a non-strict way, without the "var" directive.

**Comment by atsepkov** [comment]
Yea, I apologize for that. symbolfor is a result of a bad merge I did a while back, this function should not even exist in this version of RapydScript - it's part of kwargs handling, which I do differently from -ng fork. The reason strict complains in this case is because symbolfor is not prefixed with a `var ` keyword, it thinks that we're attempting to use the variable without declaring it.

**Comment by atsepkov** [comment]
Fixed by 1a5de4400678aea9ed14e2febd1c74e288c28a24
